Update instructions for ESPHome development on Windows (#43)
esphome/developers.esphome.io: docs/contributing/development-environment.md

Once you've pushed your branch, if you wish, you can
[submit your work for integration into ESPHome](submitting-your-work.md).

???+ Windows

    ## Testing Development Changes on Windows
    

>     On Windows, One way to test changes is by creating a branch in your remote fork of the main ESPHome GitHub repo,
>     and then installing it just like [Installing ESPHome Manually](https://esphome.io/guides/installing_esphome) with the below command:

Reviewer: ```suggestion
    To test changes when using Windows:
    - Create a branch in your remote fork of the main ESPHome GitHub repo
    - Install from your fork in the same manner you would
      [install ESPHome manually](https://esphome.io/guides/installing_esphome) but with the following command:
```

Author: never occurred to me I could totally create an UL inside collapsible.
Thanks

Reviewer: Fixed -- just needed a newline. 😅

>     Here, username and branch is your username and ESPHome fork branch name respectively on Github.

Reviewer: I think this should go below the commands, but also:
```suggestion
    Here, `username` and `branch` are (respectively) your GitHub username and branch name in your ESPHome fork on Github.
```

Once you've pushed your branch, if you wish, you can
[submit your work for integration into ESPHome](submitting-your-work.md).

???+ Windows

    ## Testing Development Changes on Windows
    

>     To test changes when using Windows:

Reviewer: ```suggestion
    To test changes when using Windows:

```

Once you've pushed your branch, if you wish, you can
[submit your work for integration into ESPHome](submitting-your-work.md).

???+ Windows

    ## Testing Development Changes on Windows
    
    To test changes when using Windows:

    - Create a branch in your remote fork of the main ESPHome GitHub repo
    - Install from your fork in the same manner you would

>       [install ESPHome manually](https://esphome.io/guides/installing_esphome) but with the following command:
> 
>     Here, `username` and `branch` are (respectively) your GitHub username and branch name in your ESPHome fork on Github.
> 
>     ```bash
>     pip install --pre https://github.com/username/esphome/archive/branch.zip
>     ```
>     OR
>     ```bash
>     pip install -U git+https://github.com/username/esphome.git@branch --force-reinstall
>     ```
>     The `--force-reinstall` flag is important for testing small changes!

Reviewer: ```suggestion
[install ESPHome manually](https://esphome.io/guides/installing_esphome) but with one of the following commands:

        ```bash
        pip install --pre https://github.com/username/esphome/archive/branch.zip
        ```
        OR
        ```bash
        pip install -U git+https://github.com/username/esphome.git@branch --force-reinstall
        ```

    Note that `username` and `branch` are (respectively) your GitHub username and branch name in your ESPHome fork on Github.

    The `--force-reinstall` flag is important for testing changes!
```
